test/irb/test_context.rb

```diff
@@ -105,6 +105,24 @@ def test_eval_input
       $VERBOSE = verbose
     end
 
+    def test_eval_input_raise2x
```

Comment:
Would you add "skip TruffleRuby" to the test [like other tests](https://github.com/ruby/irb/blob/8c6324af3ac06159f738839df72d0ff0366975b2/test/irb/test_context.rb#L79)? Backtraces are different between CRuby and TruffleRuby.